user-guide/Advanced_Modules/User Defined APIs/UD_APIs_UserDefinableApiEndpoint.md

```diff
@@ -151,11 +153,14 @@ This section contains options specific to the DataMiner User-Definable APIs modu
 
 - **NatsSubject**: The subject used for internal NATS messaging. This should not be changed.
 
-- **SessionConfigPath**: The path to the *SLCLoud.xml* file.
+- **SessionConfigPath**: *Optional.* The path to the NATS config file. This will by default point to the `SLCloud.xml` file if not filled in.
+
+- **CredentialsConfigPath**: *Optional.* The path to the credentials used to connect to DataMiner over NATS. This will point to the default DataMiner credentials file when not filled in.
 
-- **CredentialsConfigPath**: The path to the credentials used to connect to DataMiner over NATS.
+- **MessageBrokerTimeOutSeconds**: The time the message broker sending the NATS trigger to SLNet will wait for a response before it times out. By default, this is set to 90 seconds (i.e. 1.5 minutes).
```

Comment:
S: The time the message broker (sending the NATS trigger to SLNet) will wait for a response before it times out.

Reply:
```suggestion
- **MessageBrokerTimeOutSeconds**: The time the message broker (sending the NATS trigger to SLNet) will wait for a response before it times out. By default, this is set to 90 seconds (i.e. 1.5 minutes).
```